_rules/image-button-name-descriptive-f3b293.md

```diff
@@ -146,23 +144,23 @@ The `input` element with type with a `type` attribute in the `Button` state is n
 
 #### Inapplicable Example 3
 
-The `button` element is tested separately from the `img` element. [Success Criterion 4.1.2 Name, Role, Value](https://www.w3.org/TR/WCAG21/#name-role-value) is applied to the button, whereas the image is tested under [Success Criterion 1.1.1 Non-text Content](https://www.w3.org/TR/WCAG21/#non-text-content)
+This `button` element is separate from the `img` element.
 
 ```html
 <button><img src="/test-assets/shared/search-icon.svg" alt="Search" /></button>
 ```
 
 #### Inapplicable Example 4
 
-The `img` element is not a user interface component, and so is not tested for [Success Criterion 4.1.2 Name, Role, Value](https://www.w3.org/TR/WCAG21/#name-role-value).
+This `img` element is not button component.
```

Comment:
Description of these two examples could also be in the line of "There is no input element in the Image button state" (to match applicability more closely). Likely adding the extra explanation as second sentence.

Reply:
Could you provide some suggestions so I can better understand what you mean. Thanks.

Reply:
The rule is applicable to `input` elements in the `Image Button` state. So the reason that these snippets are inapplicable is the lack of such elements. It has nothing to do with `button` elements or components. To match the example description syntax, both these should be
"There is no `input` element in the `Image Button` state." 
An extra sentence can be added. Maybe for Inapplicable Example 3 "A `button` element containing only an `img` element is not an Image Button, this rule specifically checks for `input` elements in the `Image Button` state". And for Inapplicable Example 4 "The `img` element alone is not an Image Button."